### PR TITLE
create contributor phone

### PR DESCRIPTION
**Pull request by turret-dev**
because contributors are epic 👍

### CODE REVIEW

**Comment by SunburntRock89** [comment]
There are 3 contributors. One doesn't use the bot at all. One of them is you. Perhaps you're trying to too your own horn slightly?

**Comment by turret-dev** [comment]
but we have a redphone emote anyway so

**Comment by SunburntRock89** [comment]
You didn't deny it, and we don't need to use everything we have.

**Comment by turret-dev** [comment]
i wouldn't even be able to use it because of my customer support so

**Comment by turret-dev** [comment]
also according to (this)[https://github.com/DTel-HQ/dtel/graphs/contributors] we have 21 contributors including the dtel bosses and rexo

**Comment by SunburntRock89** [comment]
The people who have the role are the people who we care about here. If they don't have the role, they don't get the phone, and also probably don't care about the bot anymore

**Comment by Rexogamer** [comment]
Sun did you fucking listen to Mitch's idea

Turret, I ❤️ this

File: Internals/callHandler.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ module.exports = async(cmd, msg, suffix, call) => {
 	let phone = config.callPhones.default;
 	if (fromvip) phone = config.callPhones.donator;
 	if (!hidden) {
+		if (msg.author.contributor) phone=config.callPhones.contributor;
```
Comment (Rexogamer) [inline Internals/callHandler.js:26]: Minor spacing error
```suggestion
		if (msg.author.contributor) phone = config.callPhones.contributor;
```

**Comment by Mitchell3514** [comment]
This wasn't exactly my idea. I thought the red phone was meant as a boss phone (which I also don't support, but don't mind either). 
Sun is definitely right, but also why not tbh.

**Comment by SunburntRock89** [comment]
Have you run this through ESLint? Also if you're gonna let this through, why don't you just go for the full shebang and make one for server boosters or something?

**Comment by turret-dev** [comment]
boss phone would be like dark blue since their role color is blue

File: Structures/DTelUser.js [hunk 2]
```diff
@@ -67,7 +68,7 @@ module.exports = Discord => {
 					boss: this.boss,
 					manager: this.manager,
 					support: this.support,
-					donator: this.donator,
+					donator: this.donator
```
[the remark this answers is not in the document]
Comment (SunburntRock89) [reply Structures/DTelUser.js:71]: agreed

**Comment by Rexogamer** [comment]
ideas:
- pink one for boosters
- normal blue phone but with like yellow stripes for boss?
- orange for supervisor (required ASAP)

**Comment by SunburntRock89** [comment]
And just like that, you've eliminated your chances of getting a supervisor badge

**Comment by Rexogamer** [comment]
they were just ideas, who burnt you

**Comment by austinhuang0131** [comment]
boosters should not get badges because we don't get any benefits

**Comment by turret-dev** [comment]
this is just a commit about making a red phone for contributors, not fucking giving every single person a unique phone

**Comment by austinhuang0131** [comment]
reminder that this is a public repo

no profanity

**Comment by turret-dev** [comment]
shit i broke the rules fuck

**Comment by turret-dev** [comment]
mitchell, fyi the reason i removed the comma is because i added a contributor perm but then i was like "nah, contributor only commands aint comin" so then i removed it and didnt know that eslint gets picky with commas so i sowwy

**Review by Rexogamer — APPROVED** [review]
back on topic, this lgtm code wise (although maybe the red phone should be boss instead?)

**Review by SunburntRock89 — CHANGES_REQUESTED** [review]
Have you actually ran this and made sure it worked?

**Comment by Rexogamer** [comment]
> Have you actually ran this and made sure it worked?

code wise this is basically the same as the other role phones, although whether it actually shows is another matter since CS is above it (out of scope but what if a user could have multiple phones showing)

**Comment by SunburntRock89** [comment]
> out of scope but what if a user could have multiple phones showing)
No, that'd just be cluttered

**Comment by Mitchell3514** [comment]
I'm not allowing us to lose our A+ on codacy, fix your code.

**Comment by Rexogamer** [comment]
> I'm not allowing us to lose our A+ on codacy, fix your code.

all good now

**Comment by Mitchell3514** [comment]
This hound thing seems useless

**Pull request merged**